src/main/java/org/embulk/output/s3/S3FileOutputPlugin.java

```diff
@@ -140,6 +145,8 @@ private AmazonS3 newS3Client(final PluginTask task)
         {
             Optional<String> endpoint = task.getEndpoint();
             Optional<String> region = task.getRegion();
+            Optional<Boolean> path_style_access = task.getPathStyleAccess();
+
```

Comment:
We use the camelCase for variable names.

Also, an unnecessary newline.

```suggestion
            final boolean usePathStyleAccess = task.getUsePathStyleAccess();
```